### PR TITLE
Hopefully fix righhtpanel crash

### PR DESCRIPTION
**Pull request by dbkr**
The right panel wears many hats, and only one of those hats comes
with a room prop, so make sure it's there.

### CODE REVIEW

File: src/components/structures/RightPanel.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ Copyright 2015, 2016 OpenMarket Ltd
 Copyright 2017 Vector Creations Ltd
 Copyright 2017, 2018 New Vector Ltd
 Copyright 2019 Michael Telatynski <7t3chguy@gmail.com>
-Copyright 2019 The Matrix.org Foundation C.I.C.
+Copyright 2019, 2020 The Matrix.org Foundation C.I.C.
```
Comment (turt2live) [inline src/components/structures/RightPanel.js:6]: While you're here, fancy converting those Vector people's copyrights to the foundation? Michael's would remain, but Vector Creations and New Vector both should have been converted to the Foundation at some point (but we didn't)

**Review by turt2live — APPROVED** [review]

File: src/components/structures/RightPanel.js (hunk 1 above)
Comment (dbkr) [reply src/components/structures/RightPanel.js:6]: hmm - that wasn't my understanding of how it worked? I didn't think we ever updated copyrights retrospectively.

Comment (turt2live) [reply src/components/structures/RightPanel.js:6]: indeed, but a clarification a while ago lead to (painfully) slow adoption of fixing them: https://github.com/vector-im/element-web/issues/13936

Comment (dbkr) [reply src/components/structures/RightPanel.js:6]: huh, TIL

**Pull request merged**